apis.md

```diff
@@ -9,3 +9,10 @@ The following OpenTracing APIs are currently available in the RFC stage:
 
 Please refer to the README files in the respective projects for examples of usage.
 
+### Implementation Notes
+
+OpenTracing APIs intend to express a similar experience to the user, while still being idiomatic to the language or framework. The following concepts apply to all variants.
+
+A **Span** is shared mutable state. It accumulates tags and logs, and can also create children. Finishing a span means capturing its state as a SpanRecord.
+
+A **SpanRecord** is an immutable representation of a completed span. It can be reported directly to an OpenTracing system, or encoded into a representation of an alternate system, such as HTrace or Zipkin.
```

Comment:
>
> I agree, implementations of the API should be discussed in a separate area.
>
> I third the notion!

> For the v1.0 API I'm less and less confident that we should really talk
> about the "standard tracer"
>
> Do you still think we should provide one in the API projects? I got
> distracted today by the API harness (which already paid off once I ran our
> real impl against it), so haven't started on the standard tracer for
> Python.
>
I'm not sure, but seems if it were called Mock or FakeTracer, it would
still allow folks to playground, and write test harnesses, but without the
warranties.

> Perhaps instead of doing that I could fix the zipkin-like to be testable.
> We could also move zipkin-like out into its own repo, and actually call
> it python-zipkin, as in the "reference implementation". Problem is it
> doesn't have a real reporter to submit to Zipkin.
>
Other tracers for zipkin exist in the OpenZipkin org, and have coupling
there. I think it would make more sense for a new python zipkin tracer to
exist as "zipkin-python" or similar there.. @abesto thoughts?